assets/translations.json

```diff
@@ -32,24 +32,19 @@
         "dataForGood": "<p>Our team has gathered significant expertise in interactive data visualization, machine learning, and developing artificial intelligence solutions. We've decided to put these technologies to work in resolving the world's most pressing challenges.</p> <p>In our Data for Good initiative, we focus primarily on climate change and environmental protection projects, because we believe these issues will become the primary challenges to humanity in the decades to come.</p>"
       },
       "tools": {
-        "title": "Media tools",
-        "mode": "Mode of operation",
-        "extractFramesDetail": "Extract frames from videos at regular intervals",
-        "createThumbnailsDetail": "Create a thumbnail for each image / video",
-        "extractionInterval": "Interval between frames in seconds",
+        "title": "Create thumbnails",
         "thumbnailSize": "Maximum thumbnail width/height in pixels",
         "thumbnailSizeSmall": "Small",
         "thumbnailSizeMedium": "Medium",
         "thumbnailSizeLarge": "Large",
         "chooseInput": "Choose input directory",
         "chooseOutput": "Choose output directory",
-        "extractFrames": "Extract frames",
         "createThumbnails": "Create thumbnails",
         "logTitle": "Tool output",
         "success": "Image extraction completed.",
         "failure": "Image extraction failed.",
         "inProgress": "Please wait. Image extraction in progress. This may take a long time.",
-        "info": "<p>Please select mode of operation. You can extract frames from videos to later detect animals using the Detect animals tab (if you are not using videos, go directly to the Detect animals tab) or create thumbnails for all videos and images.</p> <p>Please select an input directory containing subfolders with media (recognized formats: avi, mp4, jpg, png).</p> <p>Please select an output directory where the folder structure from the input directory will be replicated and the images will be stored.</p>"
+        "info": "<p>You can create thumbnails for all videos and images.</p> <p>Please select an input directory containing subfolders with media (recognized formats: avi, mp4, jpg, png).</p> <p>Please select an output directory where the folder structure from the input directory will be replicated and the images will be stored.</p>"
```

Comment:
Creating thumbnails for videos didn't work for me (I checked some .avi and .mp4 samples). For .jpg and .png it works fine though

Reply:
Forgot to update the english translation, thanks!